pkg/hostagent/requirements.go

```diff
@@ -132,6 +132,20 @@ Also see "/var/log/cloud-init-output.log" in the guest.
 A possible workaround is to run "lima-guestagent install-systemd" in the guest.
 `,
 	})
+	if *a.y.SSH.ForwardAgent {
+		req = append(req, requirement{
+			description: "linking ssh auth socket to /tmp/ssh-auth/agent.sock",
+			script: `#!/bin/bash
+set -eux -o pipefail
+if ! timeout 30s bash -c "mkdir -p -m 700 /tmp/ssh-auth && ln -f $SSH_AUTH_SOCK /tmp/ssh-auth/agent.sock"; then
+	echo >&2 "unable to link $SSH_AUTH_SOCK to /tmp/ssh-auth/agent.sock"
+	exit 1
+fi
+`,
+			debugHint: `Linking $SSH_AUTH_SOCK to /tmp/ssh-auth/agent.sock failed.
+Ensure the SSH_AUTH_SOCK environment variable exists and is not empty.`,
```

Comment:
This is not really helpful, as the user wouldn't know how to create the variable; it is provided by `ssh`, and if it isn't created, I'm not sure how the user can fix it.

In practice I think this will never happen; when the ssh connection succeeds, and agent forwarding has been enabled, then ssh will create the variable.

Reply:
Yeah I'm not really sure what helpful advice to put here other than it failed

Reply:
> Yeah I'm not really sure what helpful advice to put here other than it failed

I think that is enough. As I said above, I expect this to be essentially unreachable code.

Reply:
Ok removed the unhelpful messaging.